data/35e/wizards_of_the_coast/supplement/lords_of_madness/lom_classes.lst

```diff
@@ -139,7 +139,7 @@ CLASS:Topaz Guardian	DEFINE:TopazGuardialLVL|0	BONUS:VAR|TopazGuardialLVL|CL
 CLASS:Topaz Guardian	STARTSKILLPTS:2
 CLASS:Topaz Guardian	CSKILL:Climb|Concentration|TYPE.Craft|Intimidate|Jump|Knowledge (Dungeoneering)|Knowledge (Religion)|TYPE.Profession|Sense Motive|Spot
 1	ABILITY:Special Ability|AUTOMATIC|Topaz Diadem
-2	ABILITY:Special Ability|AUTOMATIC|Smite Aberration		ADD:SPELLCASTER|ANY
+2	ABILITY:Special Ability|AUTOMATIC|Smite Aberration ~ Topaz Guardian			ADD:SPELLCASTER|ANY
```

Comment:
Key standard here should be Name of Grantor ~ Class Feature Name, Topaz Guardian ~ Smite Aberration.